### PR TITLE
docs: playground links

### PR DESCRIPTION
**Pull request by nnaydenow**
With this change all links are working correctly in GitHub. In addition links are preprocessed to the correct format to work after Jekyll playground is served.

Following link patterns where tested:
#test
./01-first-steps.md
./01-first-steps.md#test
../1-getting-started/01-first-steps.md
../1-getting-started/01-first-steps.md#test
./README.md
../1-getting-started/README.md

Fixes: #6407

### CODE REVIEW

**Review by vladitasev — APPROVED** [review]
`docs/test` are intended to be merged, right?

**Comment by nnaydenow** [comment]
> Referen

There

> `docs/test` are intended to be merged, right?

Broken links inside README.md are not handled and this should be addressed to so the PR is still not for merge. About your question I've cleaned it from docs folder but forgot to remove it after building it into the playground.

**Comment by nnaydenow** [comment]
Hi @vladitasev,

Test directory appears only into documentation in root directory so we can test links whenever we need that.

To test the links we have to exclude the "test" folder from the list with files/folders that have to be skipped.

File: packages/playground/package.json [hunk 1]
```diff
@@ -57,12 +57,6 @@
     "prepare:assets": "npm run clean:assets && npm run copy:assets",
     "prepare:samples": "npm run clean:samples && npm run copy:samples",
     "prepare:pages": "npm run clean:pages && node ./build-scripts/pages-prepare.js",
-    "prepare:documentation": "node ./build-scripts/documentation-prepare.js \"../../docs/\" \"./docs/\"",
-    "prepare:next:nojekyll": "copy-and-watch \"./.nojekyll\" \"./dist\"",
-    "prepare:next:samples": "node ./build-scripts-storybook/samples-prepare.js",
-    "prepare:next:manifest": "node ./build-scripts-storybook/parse-manifest.js",
-    "prepare:next:documentation": "node ./build-scripts-storybook/documentation-prepare.js",
-    "storybook": "npm-run-all --parallel prepare:next:* && storybook dev -p 6006",
```
Comment (vladitasev) [inline packages/playground/package.json:65]: are these 2 intended to be removed?

Comment (nnaydenow) [reply packages/playground/package.json]: Restored.

**Pull request merged**